package.json

```
@@ -49,6 +49,9 @@
     "template-starter-typescript:build": "pnpm --filter @commercetools-applications/merchant-center-template-starter-typescript run build",
     "template-starter-typescript:start": "pnpm --filter @commercetools-applications/merchant-center-template-starter-typescript run start",
     "template-starter-typescript:start:prod:local": "pnpm --filter @commercetools-applications/merchant-center-template-starter-typescript run start:prod:local",
+    "template-custom-view-starter-javascript:build": "pnpm --filter @commercetools-applications/merchant-center-custom-view-template-starter run build",
+    "template-custom-view-starter-javascript:start": "pnpm --filter @commercetools-applications/merchant-center-custom-view-template-starter run start",
+    "template-custom-view-starter-javascript:start:prod:local": "pnpm --filter @commercetools-applications/merchant-center-custom-view-template-starter run start:prod:local",
```

Nit: names seem to be a bit inconsistent with the Custom Applications

```suggestion
    "template-custom-view-starter:build": "pnpm --filter @commercetools-applications/merchant-center-custom-view-template-starter run build",
    "template-custom-view-starter:start": "pnpm --filter @commercetools-applications/merchant-center-custom-view-template-starter run start",
    "template-custom-view-starter:start:prod:local": "pnpm --filter @commercetools-applications/merchant-center-custom-view-template-starter run start:prod:local",
```

---

I agree they are inconsistent right now as I didn't want to change anything related to Custom Applications just in case there were any unexpected impact.

From my point of view, Custom Application commands are the ones who lack a more meaningful name and what I think it would have more sense is something like this:
```
"template-custom-application-starter-javascript:build"
"template-custom-application-starter-javascript:start"
"template-custom-application-starter-javascript:start:prod:local"
"template-custom-application-starter-typescript:build"
"template-custom-application-starter-typescript:start"
"template-custom-application-starter-typescript:start:prod:local"

"template-custom-view-starter-javascript:build"
"template-custom-view-starter-javascript:start"
"template-custom-view-starter-javascript:start:prod:local"
"template-custom-view-starter-typescript:build"
"template-custom-view-starter-typescript:start"
"template-custom-view-starter-typescript:start:prod:local"
```

Does it make sense to you our do you prefer to infer javascript is the default implementation?

---

I see, thanks for the clarification.

From my experience using other tools and libraries, JS is normally the default implementation yes. However, I do see your point and I don't mind either way. For me it's more important to have it consistent.

Maybe you can run a quick poll in the team to get their feedback as well. Renaming the Custom Applications template then should be a follow up.
Thanks

---

I updated to default names to Javascript here: adc437d55708f9611433001be0dcff3bad79f6f8

```
"template-custom-application-starter:build"
"template-custom-application-starter:start"
"template-custom-application-starter:start:prod:local"
"template-custom-application-starter-typescript:build"
"template-custom-application-starter-typescript:start"
"template-custom-application-starter-typescript:start:prod:local"

"template-custom-view-starter:build"
"template-custom-view-starter:start"
"template-custom-view-starter:start:prod:local"
"template-custom-view-starter-typescript:build"
"template-custom-view-starter-typescript:start"
"template-custom-view-starter-typescript:start:prod:local"
```